package.json

```diff
@@ -1,44 +1,18 @@
 {
-  "name": "replicache-examples",
+  "name": "replicache-quickstarts",
   "version": "0.1.0",
   "private": true,
-  "dependencies": {
-    "dotenv": "^16.0.1",
-    "express": "^4.18.1",
-    "replicache": ">=11.2.1",
-    "replicache-express": "^0.1.4"
-  },
   "devDependencies": {
     "@rocicorp/eslint-config": "^0.1.2",
     "@rocicorp/prettier-config": "^0.1.1",
-    "@types/express": "^4.17.13",
-    "@types/node": "^16.11.50",
-    "concurrently": "^7.3.0",
-    "nodemon": "^2.0.19",
     "npm-run-all": "^4.1.5",
-    "prettier": "^2.2.1",
-    "ts-node": "^10.9.1",
     "typescript": "4.7.4"
   },
   "scripts": {
     "format": "prettier --write './**/src/**/*.{js,jsx,json,ts,tsx,html,css,md}' '*.{cjs,js,jsx,json,ts,tsx,html,css,md}'",
     "check-format": "prettier --check './**/src/*/*.{js,jsx,json,ts,tsx,html,css,md}' '*.{cjs,js,jsx,json,ts,tsx,html,css,md}'",
     "lint": "eslint --ext .ts,.tsx,.js,.jsx .",
-    "postinstall": "cd ./react/ && npm install && cd ../ts-web-component && npm install",
-    "build": "npm run lint && run-p build:*",
-    "build:server": "rm -rf lib && tsc",
-    "build:react": "cd ./react/ && rm -rf lib && rm -rf dist && npm run build",
-    "build:ts-web-component": "cd ./ts-web-component/ && rm -rf lib && rm -rf dist && npm run build",
-    "typecheck": "tsc -b",
-    "tsc-version": "tsc -v",
-    "dev:react": "cd ./react && npm run dev",
-    "dev:ts-web-component": "cd ./ts-web-component && npm run dev",
-    "dev:server": "nodemon",
-    "watch:react": "concurrently --kill-others 'npm run dev:server' 'npm run typecheck -- --watch' 'npm run dev:react'",
-    "watch:ts-web-component": "concurrently --kill-others 'npm run dev:server' 'npm run typecheck -- --watch' 'npm run dev:ts-web-component'",
-    "clean:dist": "rm -rf ./dist && mkdir -p ./dist",
-    "prod:react": "npm run clean:dist && cp -r ./react/dist/* ./dist/ && NODE_ENV=production node --loader ts-node/esm --experimental-specifier-resolution=node ./server/src/server.ts",
-    "prod:ts-web-component": "npm run clean:dist && cp -r ./ts-web-component/dist/* ./dist/ && NODE_ENV=production node --loader ts-node/esm --experimental-specifier-resolution=node ./server/src/server.ts"
+    "build": "npm run build -ws"
```

Comment:
suggest replacing this with a 

`check-types` that does `npm run check-types -ws`, and the individual package `check-types` can do `tsc --noEmit`